recws.go

```diff
@@ -91,6 +93,17 @@ func (rc *RecConn) Close() {
 	rc.setIsConnected(false)
 }
 
+// Shutdown gracefully closes the connection by sending the websocket.CloseMessage.
+func (rc *RecConn) Shutdown() {
```

Comment:
would be great to have the `writeWait` as  `Shutdown` param instead of a global const